### ISSUE TITLE
README example wrong

### ISSUE THREAD

**Issue by tomjaguarpaw** [jfischoff/tmp-postgres#19]
The README gives the following as an example

```haskell
result <- start []
case result of
  Left err -> print err
  Right tempDB -> do
     -- Do stuff
     stop tempDB
```

but it seems it should be more like

```haskell
result <- start defaultOptions
case result of
  Left err -> print err >> pure Nothing
  ...
```

**Comment by jfischoff** [comment]
Yeah. The readme needs to be updated.

**Comment by jfischoff** [comment]
Fixed with version update to 1.0.0.0
